src/pages/workspaces/workspace/analysis/modals/persistent-disk-controls.test.ts

```diff
@@ -20,16 +20,17 @@ import {
   defaultNumDataprocWorkers,
   defaultNumGpus,
   defaultPersistentDiskType,
-  getDefaultMachineType,
+  getDefaultMachineType
 } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
 import { toolLabels } from 'src/pages/workspaces/workspace/analysis/tool-utils'
 
 
 const defaultIComputeConfig = {
-  selectedPersistentDiskSize: defaultGcePersistentDiskSize,
-  selectedPersistentDiskType: defaultPersistentDiskType,
+  persistentDiskSize: defaultGcePersistentDiskSize,
+  persistentDiskType: defaultPersistentDiskType,
   masterMachineType: getDefaultMachineType(false, toolLabels.RStudio),
   masterDiskSize: defaultDataprocMasterDiskSize,
+  diskSize: 0, //TODO: set to a default disk size for Azure
```

Comment:
50 seems to be our default

Reply:
realized this was leftover from before we changed diskSize --> persistentDiskSize, so I nixed it